app/src/main/java/com/example/soptexampleproject/presentation/home/screens/ViewPagerActivity.kt

```diff
@@ -0,0 +1,61 @@
+package com.example.soptexampleproject.presentation.home.screens
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import androidx.fragment.app.Fragment
+import androidx.viewpager2.widget.ViewPager2
+import com.example.soptexampleproject.R
+import com.example.soptexampleproject.databinding.ActivityViewPagerBinding
+import com.example.soptexampleproject.presentation.home.screens.fragment.FragmentChangeAdapter
+import com.example.soptexampleproject.week3.Fragment.PagerFragmentList
+import com.example.soptexampleproject.week3.Fragment.PagerFragmentProfile
+import com.example.soptexampleproject.week3.Fragment.PagerFragmentSetting
+
+class ViewPagerActivity : AppCompatActivity() {
+    private lateinit var adapter: FragmentChangeAdapter
+    private val myFragments =
+        listOf<Fragment>(PagerFragmentProfile(), PagerFragmentList(), PagerFragmentSetting())
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        ActivityViewPagerBinding.inflate(layoutInflater).run {
+            setContentView(root)
+            bindingView(this)
+        }
+    }
```

Comment:
```suggestion
    private lateinit var binding: ActivityViewPagerBinding
    private lateinit var adapter: FragmentChangeAdapter
    private val myFragments =
        listOf<Fragment>(PagerFragmentProfile(), PagerFragmentList(), PagerFragmentSetting())

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = DataBindingUtil.setContentView(this, R.layout.activity_view_pager)
        
        bindingView()
    }
```

이미 알고계시겠지만 그냥은 못넘어ㄱㅏ...
전역변수 binding을 추가하면 **bindingView()** 에 **binding**을 인자로 넘길 필요가 없습니다~!

Reply:
이부분이 github android 참고해서 만들었었는데, binding 변수 자체를 전역으로 안 가지고 있을 수가 있더라구요. 그래서 run으로 나온 binding 객체를 함수로만 보내서 적용하는걸 해봤는데 별로인가요??

Reply:
그 예제에서도 binding 다른 함수의 인자로 넘겼나요..?
onCreate에서만 binding을 초기화하고 끝나면 전역변수가 굳이 필요없겠지만,, 다른 함수에서도 binding을 쓴다면 전역변수가 좋을 것같아요!

Reply:
WorkManager 예제인데 거기서도 다른 함수에서 따로 안 써서 해당 방법으로 보낸것같네요! 좋습니당!!!